### PR TITLE
API pull Request

### PR DESCRIPTION
**Pull request by mwanawabangona**
# In this feature update:

 - Use Gitflow
- Create an npm project with webpack.
- Write plain HTML markup with minimum styling using project wireframe as a guide.

### CODE REVIEW

File: src/listcore.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+//   const scoreBoardContainer = document.querySelector('#scores');
+//   scoreBoardContainer.innerHTML = '';
+//   listscore.forEach((user) => {
+//     scoreBoardContainer.innerHTML += `<li>
+//         <p>${user.name}: ${user.score}</p>
+//         </li>`;
+//   });
+// };
+
+// exports.displayScores = displayScores;
```
Comment (rloterh) [inline src/listcore.js:62]: - [ ] OPTIONAL
There are comments found in your code on the repository. Kindly endeavor to remove **all**  block comments from your code before pushing it to the repository so as to keep your code well organized.

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Ellis,  

Your project is complete! Great job for nailing it right on the first attempt 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍

- JavaScript promise feature well implemented
- Data is sent and received over the API successfully
- Module import correctly implemented
- ES6 Syntax used
- No linter errors

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly consider my comments marked **[OPTIONAL]** under this review:_
<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — APPROVED** [review]
Hi Ellis, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 :
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/listcore.js (hunk 1 above)
Comment (mwanawabangona) [reply src/listcore.js:62]: @rloterh thank you for your review

**Pull request merged**